src/Logic.hs

```diff
@@ -161,7 +161,10 @@ runAction config = foldFree $ \case
     Git.forcePush sha prBranch >>
     Git.tag' sha newTag >>=
     \case TagFailed _ -> cont . (Left "Please check the logs",) <$> Git.push sha (Git.Branch $ Config.branch config)
-          TagOk tagName -> cont . (Right tagName,) <$> Git.pushAtomic [AsRefSpec tagName, AsRefSpec (sha, Git.Branch $ Config.branch config)]
+          TagOk tagName -> cont . (Right tagName,) <$>
+          -- Deleting tag after atomic pushg is important to maintain one "source of truth", namely
+          -- - the origin
+            Git.pushAtomic [AsRefSpec tagName, AsRefSpec (sha, Git.Branch $ Config.branch config)] <* Git.deleteTag tagName
```

Comment:
Nit: Can you format it as 

```haskell
cont . (Right tagName,)
<$> Git.pushAtomic [AsRefSpec tagName, AsRefSpec (sha, Git.Branch $ Config.branch config)]
<*  Git.deleteTag tagName
```
?

That would make it easier to see at a glance what is happening here, I think.

Reply:
Fixed